exts/fun.py

```diff
@@ -278,21 +278,22 @@ async def type(self, ctx: commands.Context, *, name: str):
 
         `name` here is the name of the type you wish to see the details, for example: fire, electric, water etc.
         """
-        typeobj = await self.pokeclient.fetch_pokemon_type(name.lower())
-        if typeobj:
-            pokeobj = await self.pokeclient.fetch_pokemon(random.choice(typeobj.pokemon))
-            e = discord.Embed(title=f"{typeobj.name.capitalize()} type details",
-                              description=f"Example pokemon: **{pokeobj.name.capitalize()}**",
-                              colour=discord.Colour.random())
-            e.set_thumbnail(url=pokeobj.official_artwork)
-            for k, v in typeobj.damage_relations.items():
-                if v:
-                    e.add_field(name=(" ".join(k.split('_'))).capitalize(),
-                                value=(", ".join(v)).capitalize(),
-                                inline=True)
-            await ctx.send(embed=e)
+       
+        pokeobj = await self.pokeclient.get_pokemon(name)
+        if type(pokeobj) is list:
```

Comment:
This has been deprecated, now raises `NoSingleMatch` if fuzzy search couldn't pinpoint a single entry with > 80% match